lib/fast_rcnn/train.py

```diff
@@ -15,7 +15,11 @@
 import os
 
 from caffe.proto import caffe_pb2
-import google.protobuf as pb2
+
+# Importing google.protobuf and accessing text_format from it doesn't work,
+# but this does.
+import google.protobuf.text_format as pb2_text_format
```

Comment:
This works fine with older versions of the `protobuf` library
`protobuf>=2.5.0` and `<=2.6.1`
I modified the requirements file in our `caffe` fork to get around this issue (and some other issues with newer versions of `numpy`).
https://github.com/kinsolresearch/caffe/commit/28529bd0c6b21e522b58931ae4c31a3bcf3a5b87

I was trying to avoid having to modify and maintain this library, but since it looks like you fixed these issues, should we revert my commits so this will use the latest versions?